src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4 px rgba(0 0 0 0.05);
```

Comment:
There is a syntax error in the `box-shadow` property value. The color function rgba should have commas separating the values.